src/main/fc/fc_core.c

```diff
@@ -627,9 +626,8 @@ static void subTaskMotorUpdate(void)
     }
 #endif
 
-    if (motorControlEnable) {
-        writeMotors();
-    }
+    writeMotors();
```

Comment:
While I am quite happy with this change, looking at it from the Dshot command angle (and the need for Dshot commands to not be interrupted by motor updates), I am wondering if the following should be guarded by `if (pwmAreMotorsEnabled())` as well:
https://github.com/martinbudden/cleanflight/blob/71ba52bd340b80a392cfbf51cba1b855161a430e/src/main/flight/mixer.c#L474